src/control/settings/Settings.cpp

```diff
@@ -48,6 +48,14 @@ void Settings::loadDefault()
 	this->showTwoPages = false;
```

Comment:
Rename this variable to something like `showPagesInColumns` as it is the flag for all columns views.

Reply:
All of the changes related to this are in the latest of two commits.
All "Two Pages" have been changed to a more accurate "Paired Pages".
.xournalpp/toolbar.ini file with old "TWO_PAGES" doesn't seem to cause any problems (?).